Provenance socioeconomics (#303)
elixir-europe/infectious-diseases-toolkit: provenance/socioeconomic-data.md

---
title: Socioeconomic data

> description: Tracking socioeconomics data and analysis steps done to them.

Reviewer: ```suggestion
description: Tracking socioeconomic data and documenting the analysis steps applied to it.
```

> description: Tracking socioeconomics data and analysis steps done to them.

Reviewer: ```suggestion
description: Tracking socioeconomic data and documenting the analysis steps applied to it.
```
